WordPress/wordcamp.org
# QuickBooks: OAuth2 authentication flow

Our current QuickBooks authentication uses OAuth 1.0, however Intuit is deprecating that method on December 17th in favor of OAuth 2.0. This adds tools to authenticate with OAuth 2.0, using Intuit's QuickBooks V3 PHP SDK. This will also set us up for using more of the SDK's functionality in the future for improvements to our payments and reporting.

Fixes #283 

**To test**

* Testing will probably only work on the sandbox since the Redirect URI is hard coded in the app created on the Intuit site.
* Make sure your sandbox is up-to-date so that it has the SDK installed.
* Navigate to Network Admin->Settings->QuickBooks
* It may already be connected. Try both connecting and disconnecting.
* When connecting, use our WordCamp Developer credentials to log into the Intuit site.
